hdate/date.py

```diff
@@ -94,6 +126,31 @@ def __ge__(self, other: "HDate") -> bool:
         """Implement the greater than or equal operator."""
         return not self < other
 
+    def get_hebrew_number(self, number: int, short: bool = False) -> str:
+        """Get the number representation based on the current languageuage."""
+        hstring: str = hebrew_number(num=number, language=self.language, short=short)
+        return hstring
+
```

Comment:
I understand the reason for wanting this helper method, but HDate is already big as is. Please remove. You can use `hebrew_number` directly.